libraries/AC_WPNav/AC_WPNav.cpp

```diff
@@ -640,23 +633,67 @@ float AC_WPNav::get_yaw() const
     }
 }
 
+// returns target yaw rate in centi-degrees / second (used for wp and spline navigation)
+float AC_WPNav::get_yaw_rate() const
```

Comment:
```suggestion
float AC_WPNav::get_yaw_rate_cds() const
```

... the setter includes `_cds`

Reply:
I was thinking we should be consistent with the get_yaw() that is right above it and does not include "_cd()".. but you're right, we should include these non-standard units where possible.  Also change the member name.